src/v2/ion/IonResponseHelper.js

```diff
@@ -152,18 +152,18 @@ const isIonErrorResponse = (response = {}) => {
   return response.version;
 };
 
-const isTerminalError = (res = {}) => {
-  if (!Array.isArray(res.remediations) || res.remediations.length !== 1) {
-    return false;
-  }
-
-  const name = res.remediations[0].name;
+const isTerminalError = (formName) => {
   // TODO: use regex on name?
-  return name === FORMS.TERMINAL || name === FORMS.DEVICE_ENROLLMENT_TERMINAL;
+  return formName === FORMS.TERMINAL || formName === FORMS.DEVICE_ENROLLMENT_TERMINAL;
```

Comment:
let's put terminal form names in an Array, for future cases if there are more terminal pages